pkgs/applications/window-managers/sway/wrapper.nix

```diff
@@ -9,6 +9,7 @@
 
 , enableXWayland ? true
 , dbusSupport ? true
+, name ? "sway"
```

Comment:
A PR to add `volare` should not be modifying other window managers' packages.  What is going on here?

Reply:
Most likely this package should be implemented by using `.overrideAttrs` on `pkgs.sway`.

Reply:
That was my initial approach, too, but I didn't find a satisfying way to achieve it. Will give it another go.